src/main/java/com/shmkane/sellstick/PlayerListener.java

```diff
@@ -122,14 +119,14 @@ public int getUsesFromLore(List<String> lores) {
         try {
             min = hold.get(0);
         } catch (Exception ex) {
-            SellStick.log.severe(StickConfig.instance.durabilityLine + "");
-            SellStick.log.severe("The problem seems to be that your sellstick useline number has changed.");
-            SellStick.log.severe(ex.toString());
+            plugin.getLogger().severe(plugin.getStickConfig().durabilityLine + "");
+            plugin.getLogger().severe("The problem seems to be that your sellstick useline number has changed.");
+            plugin.getLogger().severe(ex.toString());
         }
 
-        for (int i = 0; i < hold.size(); i++) {
-            if (hold.get(i) < min) {
-                min = hold.get(i);
+        for (int i : hold) {

```

Comment:
I'd wanna test this out - been a while since this method was written and I'm not sure if we can just swap to checking index vs value at that index.

Reply:
In short, this method was used in case someone wanted to write:

"%uses% out of 50 uses remain on your sellstick", or even worse, something like
"out of 50, you have %uses% uses remaining"

Reply:
That one was actually an IntelliJ suggestion

Reply:
If you think about it, it's still doing the same thing. It's going: for every item in the list hold, if the item is less than min, make the min value the value of the item.

Reply:
I've done some testing on my end with this code, and both come out with the same result every time. (list of integers, 1-10, and min value of 5, both return 1)